include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge_sort.h

```diff
@@ -16,12 +16,15 @@
 #ifndef _ONEDPL_PARALLEL_BACKEND_SYCL_MERGE_SORT_H
 #define _ONEDPL_PARALLEL_BACKEND_SYCL_MERGE_SORT_H
 
-#include <cmath>     // std::log2
-#include <limits>    // std::numeric_limits
-#include <cassert>   // assert
-#include <utility>   // std::swap
-#include <cstdint>   // std::uint32_t, ...
-#include <algorithm> // std::min
+#include <bit>         // std::bit_floor
```

Comment:
Should we switch this include for `../../utils.h` since that is where `__dpl_bit_floor` is defined?

Reply:
`../../utils.h` is included now.